subprojects/core-api/src/main/java/org/gradle/api/provider/MapProperty.java

```diff
@@ -38,7 +40,7 @@
  * @since 5.1
  */
 @SupportsKotlinAssignmentOverloading
-public interface MapProperty<K, V> extends Provider<Map<K, V>>, HasConfigurableValue {
+public interface MapProperty<K, V> extends Provider<Map<K, V>>, HasConfigurableValue, MapPropertyConfigurer<K, V>, ConfigurableValue<MapPropertyConfigurer<K, V>>, SupportsConvention {
```

Comment:
:nail_care: this probably needs a couple more overrides from `SupportsConvention` too.

Reply:
If you meant for `setToConvention...()`, those are being removed from the API, as suggested